sdk/storage/storage-file-share/review/storage-file-share.api.md

```diff
@@ -167,6 +167,7 @@ export interface DirectoryForceCloseHandlesHeaders {
     errorCode?: string;
     marker?: string;
     numberOfHandlesClosed?: number;
+    numberOfHandlesFailedToClosed?: number;
```

Comment:
numberOfHandlesFailedToClosed ?
rename to numberOfHandlesFailedToClose ?

Reply:
Agreed. However it comes directly from the generated code, I will make sure to correct the swagger file.